Casks/c/coconutbattery.rb

```diff
@@ -9,7 +9,11 @@
 
   livecheck do
     url "https://coconut-flavour.com/updates/coconutBattery_#{version.csv.first.major}.xml"
-    strategy :sparkle
+    strategy :sparkle do |items|
+      items
+        .reject { |item| item.short_version.to_s.include?("b") }
+        .map(&:nice_version)
+    end 
```

Comment:
Maybe you can use `channel` to get stable branch,
```suggestion
    strategy :sparkle do |items|
      items.find { |item| item.channel.nil? }&.short_version
    end
```